verify/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_path")
+
+sh_test(
+    name = "verify-crosstool",
+    srcs = ["verify-crosstool.sh"],
+    args = [
+        "$(location //tools:CROSSTOOL)",
```

Comment:
Can we make CROSSTOOL also have a .textpb extension?

Reply:
nope, it must be named `CROSSTOOL`.